src/main/scala/io/gatling/sbt/GatlingKeys.scala

```diff
@@ -39,7 +39,9 @@ object GatlingKeys {
   private val documentationReference =
     "See https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#working-with-gatling-enterprise-cloud for more information"
 
-  val enterpriseUrl = settingKey[URL]("URL of Gatling Enterprise.")
+  val enterpriseApiUrl = settingKey[URL]("URL of the Gatling Enterprise API.")
+
+  val enterpriseWebAppUrl = settingKey[URL]("URL of the Gatling Enterprise web app.")
```

Comment:
Not backwards-compatible (changes the name used in the build.sbt file), but this setting was never documented for end users.